.agents/rules/rust/rust.examples.md

```
@@ -0,0 +1,223 @@
+# Rust Rules - Examples
+
+## Principles Examples
+
+### 内部可変性禁止
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)